### DISCUSSION TITLE
Windows Icon Poll

### DISCUSSION THREAD

**Discussion by daiyam** [VSCodium/vscodium#1478] [Polls]
Vote for VSCodium's app icon on Windows!

We would like your feedback about the icon on Windows!

The vote ends the 20th April at midnight.

|                                                       `circle1`                                                       | `macos1`                                                                                                             | `nobg`                                                                                                             | `rect1`                                                                                                             |
| :-------------------------------------------------------------------------------------------------------------------: | -------------------------------------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------------------------ | ------------------------------------------------------------------------------------------------------------------- |
| <img src="https://raw.githubusercontent.com/VSCodium/icons/main/icons/linux/circle1/blue1/paulo22s.png" width="128"/> | <img src="https://raw.githubusercontent.com/VSCodium/icons/main/icons/linux/macos1/blue1/paulo22s.png" width="128"/> | <img src="https://raw.githubusercontent.com/VSCodium/icons/main/icons/linux/nobg/blue1/paulo22s.png" width="128"/> | <img src="https://raw.githubusercontent.com/VSCodium/icons/main/icons/linux/rect1/blue1/paulo22s.png" width="128"/> |

If you have suggestions, please add your background template to [icons](https://github.com/VSCodium/icons). We might run another vote before the next version/end of month.

**Comment by Darthagnon** [comment]
I like the icon as it was, so I vote for the closest to that: **circle**; don't see any reason for a change or rebranding. The current (backgroundless) icon is invisible on my blue taskbar.

**Reply by daiyam** [reply]
I've just edited the vote that another vote can happen if there are new suggestions.

Don't you also have the problem with VSCode? Since they are using background less icon on windows.

**Reply by geekley** [reply]
Yeah, at first I though square was alright, but when I tried circle I thought nope, this is better -- my muscle memory got too used to it. In fact, the association is so strong that it reminds me of back when I used SourceTree (icon reminds me of that a little) for git, before I switched to just using Git Graph inside Codium.

As for the simplification of the algae thing, I though it was strange at first, but just because it was unfamiliar -- now comparing both in the circle I see that the simpler one is definitely better.

Regarding BG suggestions, while I would be curious about how it'd look like with an hexagon or octagon or tilted square or squircle ... I'd end up preferring circle anyways, so whatever :laughing:. Though I will say, it's a bit small; it could have less empty space around it, to match the size of other circle icons like firefox, etc.

**Reply by daiyam** [reply]
> it could have less empty space around it, to match the size of other circle icons like firefox, etc.

Yes, I know. The shown image are the Linux ones (since they are `.png`)
But `.ico` are available at https://github.com/VSCodium/icons/tree/main/icons/win32 and have no space around.

Fell free to try them :wink:

**Reply by geekley** [reply]
Sorry, I commented here, but I actually meant size on Linux. It's not that big of a difference, but it's noticeable:
![sizes](https://user-images.githubusercontent.com/38332912/231693750-dc083b75-d7f7-40ad-be1e-449891549ab1.png)
But anyway, this is minor, so whatever.

**Reply by daiyam** [reply]
@geekley let's go on the Linux poll :wink:

**Reply by Darthagnon** [reply]
> I've just edited the vote that another vote can happen if there are new suggestions.
> 
> Don't you also have the problem with VSCode? Since they are using background less icon on windows.

I don't use VSCode, but from what I've seen of its logo, it is ~~a) purple, a less common window/taskbar/desktop wallpaper accent colour~~ (shows I don't use it; that's Visual Studio, not the Electron text editor), and b) marginally higher contrast. The gradient colouration on the borderless codium logo makes it quite low contrast and it fades into darker backgrounds and is almost completely invisible on blue. e.g. put a touch of skeuomorphic design and give it a 2px plastic white border, and it would be fine. Borderless Acrylic icon design doesn't stand out anywhere near as well as Aero or other skeuomorphic styles.

**Comment by Darthagnon** [comment]
NoBG also fades slightly into darker colours due to its gradient. On Windows, it will affect desktop shortcuts, too, as the default wallpapers are completely blue (Win7, Win10 old, Win10 new)

**Comment by daiyam** [comment]
Next stage at [Semi-Finale Poll #1489](https://github.com/VSCodium/vscodium/discussions/1489)
